docs/docs/guides/2_manage_recipes.md

```diff
@@ -28,7 +28,7 @@ meteor gen recipe sample -e postgres -s console -p enrich > recipe.yaml
 
 ```bash
 # validate specified recipes.
-$ meteor lint recipe.yml
+$ meteor lint recipe.yaml
 ```
```

Comment:
@ishanarya0 both `yml` or `yaml` formats are correct for yaml file extension

Reply:
Yes, it is not necessary. Suggested this change to keep the file name consistent with the previous step.